Pod/Classes/RealexComponent/HPPManager.swift

```diff
@@ -207,15 +210,15 @@ public class GenericHPPManager<T: Decodable>: NSObject, HPPViewControllerDelegat
     /// - Parameter json: The dictionary of paramaters and values to be encoded.
     /// - Returns: The data encoded HTML string representation of the paramaters and values.
     private func httpBodyWithJSON(_ json: NSDictionary) -> Data {
-
-        var parameters = [String: String]()
+        var parameters = self.additionalParameters ?? [:]
```

Comment:
Do we need a separate instance variable for this? Are we able to use `self.parameters` instead of `self.additionalParameters`? Do we need to strip out parameters with empty values?